Composer/packages/lib/shared/src/convertUtils/parseTypeToFragment.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { dialogGroups, DialogGroup } from '../viewUtils';
+import { PromptTab } from '../promptTabs';
+
+export function parseTypeToFragment(type: string, property: string): string {
+  const inputTypes = dialogGroups[DialogGroup.INPUT].types;
+  const index = inputTypes.findIndex(t => t === type);
+  if (index >= 0) {
+    if (property === 'property') return PromptTab.USER_INPUT;
+    if (property === 'prompt') return PromptTab.BOT_ASKS;
```

Comment:
I'm worried about this scaling to all fields. Not sure what to do about it, but we should start thinking on how to handle this.